app/controllers/responsible_body/mobile/bulk_requests_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ResponsibleBody::Mobile::BulkRequestsController < ResponsibleBody::BaseController
+  def new
+    @upload_form = BulkUploadForm.new
+  end
+
+  def create
+    @upload_form = BulkUploadForm.new(upload_form_params)
+
+    if @upload_form.valid?
+      # parse file and generate records
+      begin
+        @summary = importer.import! @upload_form.file.path, @user
+        render :summary
+      rescue StandardError => e
+        Rails.logger.debug(e.message)
+        @upload_form.errors.add(:upload, "There's a problem with that spreadsheet")
```

Comment:
```suggestion
        @upload_form.errors.add(:upload, "There’s a problem with that spreadsheet")
```

Should this also be in the en.yml file?

Reply:
Yes, agree. Done